exercises/ayomide_bamidele/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,138 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <map>
+#include <algorithm>
+#include <cctype>
+#include <vector>
+#include <iomanip>
+#include <utility>
+
+
+using std::string; using std::max; using std::find_if_not;
+using std::ifstream; using std::fstream;
+using std::getline; using std::map; using std::pair; using std::transform;
+ using std::endl; using std::vector;
+using std::cout; using std::find; using std::ofstream;
+using std::find_if; using std::vector; using std::setw;
+
+bool pass(string str, string sample, size_t limit);
+
+int main(){
```

Comment:
This function is way, way too long. It just about fits on my screen, but that's only because I have a 4K monitor. Consider how you can break down parts of the program into functions.